File: Famoser.MassPass.Data/Entities/Communications/Request/RefreshRequest.cs
using System.Collections.Generic;
using Famoser.MassPass.Data.Entities.Communications.Request.Base;
using Famoser.MassPass.Data.Entities.Communications.Request.Entities;

namespace Famoser.MassPass.Data.Entities.Communications.Request
{
    public class RefreshRequest : ApiRequest
    {
        public RefreshRequest()
        {
            RefreshEntities = new List<RefreshEntity>();
        }

        public List<RefreshEntity> RefreshEntities { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Request/RefreshRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 22/05/2016
 * Time: 23:01
 */

namespace Famoser\MassPass\Models\Request;


use Famoser\MassPass\Models\Request\Base\ApiRequest;

class RefreshRequest extends ApiRequest
{
    /**
     * @var Entities\RefreshEntity[] $RefreshEntities
     */
    public $RefreshEntities;
}